darkmoon/api/v1/jinja/templates/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+
+<head>
+    <style>
```

Comment:
The longer we have this css just hardcoded in the HTML like this the more annoying it will be in the future. A ticket will be made, do not push this off too long.